### ISSUE TITLE
Cannot find app in HACS

### ISSUE THREAD

**Issue by Hohenloherin** [jaddel/ResetZHASensor#1]
I have been searching for your app in HACS up and down, but I could not find it. What about this? 

Now I manually installed your app, and it work as expected. I am using ZHA now, before I used Deconz with CONBEE II and there WernerHP's app worked flawlessly. Under ZHA WernerHP's app did not perform and the time to clear the aqara motion sensors is roundabout 70s. 
I copied the code from apps/reset_zha_sensor.py in my local folder config/appdaemon/apps into the newly created file reset_zha_sensor.py, adapted the code according your instructions in config/appdaemon/apps/apps.yaml file and restarted HAOS. 

Thank you very much for your work! 
![Opera Snapshot_2023-09-03_180340](https://github.com/jaddel/ResetZHASensor/assets/85617168/2580bc50-6571-4dd8-ba93-6f113bfbb1fb)

**Comment by 10bn** [comment]
Yes, it hasn't been published to the HACS store yet. I will need to look up the process and add it at some point. For now, this workaround should help others integrate it. Thanks for sharing it!

**Comment by 21-Jailbraik-21** [comment]
Hello, I soldered the bridge.
And installed the reset app via Hacs.
But I can't get it to reset after 5 seconds.

Are there video instructions?

I don't know where the error is.

![reset 001](https://github.com/jaddel/ResetZHASensor/assets/93378208/99819abe-e844-4117-8bc4-ef6c6f16a305)
